security/src/main/scala/pl/touk/nussknacker/ui/security/accesslogic/AnonymousAccess.scala

```diff
@@ -0,0 +1,28 @@
+package pl.touk.nussknacker.ui.security.accesslogic
```

Comment:
the class is used only in the context of Authentication Manager only. Maybe we can move it to the the manager class and make it private? WDYT?

Reply:
True, I moved it and made it an  `AuthenticationManager` internal class